src/routes/routes.ts

```diff
@@ -24,13 +25,14 @@ const asyncHandler =
 
 // login route
 router.post('/login', asyncHandler(handleLogin))
-
 // create a booking
 router.post('/bookings', requiresAuthentication, asyncHandler(createBooking))
+// view all bookings
+router.get('/bookings/all', getAllBookingsController)
```

Comment:
Does this route require authentication?

Reply:
i havent tested, but otherwise lgtm

Reply:
i think it is ok to leave this without authentication since we are making the calendar (and cinnabot in the future) as public access